### PR TITLE
Move docs/examples to root directory.

### PR DESCRIPTION
**Pull request by seanhandley**
Seems a better place, it's more visible from the repo root level on Github.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
-gemfiles/Gemfile-1.9.lock
\ No newline at end of file
+gemfiles/Gemfile-1.9.lock
+.DS_Store
```
Comment (gildub) [inline .gitignore]: Some clean up needed please.

**Comment by gildub** [comment]
Yeah that's needed since the split from the core.
Just address the comment so we can merge.
Thanks

File: .gitignore (hunk 1 above)
Comment (seanhandley) [inline .gitignore]: I added this because OSX Finder does this with annoying regularity.

I'll put it in a separate PR.

**Comment by seanhandley** [comment]
Done. See #119

**Comment by gildub** [comment]
Travis fails, one of the environment fails to install vcr 3.0.2, Rubygem version needs to be bumped up (>=2.5.2)

**Pull request merged**